sql-plugin/src/main/java/com/nvidia/spark/rapids/iceberg/parquet/GpuParquet.java

```diff
@@ -69,7 +69,7 @@ public static class ReadBuilder {
     private long maxBatchSizeBytes = Integer.MAX_VALUE;
     private long targetBatchSizeBytes = Integer.MAX_VALUE;
     private boolean useChunkedReader = false;
-    private boolean useSubPageChunked = false;
+    private long maxChunkedReaderMemoryUsageSizeBytes = 0;
```

Comment:
I'm not happy with this long long name. But I don't have a better candidate for it. If you have a good name then please suggest.